privval/dash_core_signer_client.go

```
@@ -228,20 +252,23 @@ func (sc *DashCoreSignerClient) SignVote(chainID string, quorumType btcjson.LLMQ
 		return fmt.Errorf("decoding signature %d is incorrect size when signing vote : %v", len(blockDecodedSignature), err)
 	}
 
-	proTxHash, err := sc.GetProTxHash()
+	proTxHash, _ := sc.GetProTxHash()
```

don't ignore err here.